.github/workflows/labeler.yml

```diff
@@ -0,0 +1,27 @@
+name: "PR Labeler"
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: "Label PR"
+        uses: actions/labeler@v5
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Does this also remove labels if they were added because some changes initially existed in a pr but then they were removed?

Reply:
improved